src/software/pipeline/main_LdrToHdrMerge.cpp

```diff
@@ -54,7 +54,7 @@ int aliceVision_main(int argc, char** argv)
     int nbBrackets = 3;
     bool byPass = false;
     int channelQuantizationPower = 10;
-    int offsetRefBracketIndex = 0;
+    int offsetRefBracketIndex = 1000; // By default, use the automatic selection 
```

Comment:
Why does 1000 becomes the value for the automatic selection? I might have missed it, but I didn't see any reference to 1000 later on.

Reply:
1000 is an out of range value for this parameter that implies a default behavior reading the file written by the calibration node.